internal/boxcli/shell.go

```diff
@@ -47,9 +50,9 @@ func runShellCmd(cmd *cobra.Command, args []string, flags shellCmdFlags) error {
 	if err != nil {
 		return err
 	}
-
+	w := &writer.DevboxIOWriter{W: cmd.OutOrStderr(), Quiet: flags.quiet}
```

Comment:
should this be strictly `stdout` instead of either `Out or Stderr`?

I think it depends on what we decide the behavior we want:
- `quiet` suppresses output
- `quiet` redirects output to stderr

Reply:
I'll change this to pass Stderr only